src/script/browser/CheckBrowser.ts

```diff
@@ -58,49 +58,60 @@ const supportsCookies = (): boolean => {
 const supportsIndexDB = (): Promise<boolean> =>
   new Promise<boolean>((resolve, _reject) => {
     if (!('indexedDB' in window)) {
-      resolve(false);
-    } else {
-      // some versions of FF don't allow access to IndexDB in the private mode
-      if (navigator.userAgent.toLowerCase().indexOf('firefox') !== -1) {
-        let dbOpenRequest: IDBOpenDBRequest;
+      return resolve(false);
+    }
 
-        try {
-          dbOpenRequest = window.indexedDB.open('test');
-        } catch (error) {
-          return resolve(false);
-        }
+    if (navigator.userAgent.toLowerCase().indexOf('firefox') === -1) {
+      return resolve(true);
+    }
 
-        const connectionTimeout = setTimeout(() => resolve(false), 10000);
-        dbOpenRequest.onerror = event => {
-          clearTimeout(connectionTimeout);
-          if (dbOpenRequest.error) {
-            event.preventDefault();
-            return resolve(false);
-          }
-          return undefined;
-        };
-        dbOpenRequest.onsuccess = _event => {
-          clearTimeout(connectionTimeout);
-          resolve(true);
-        };
-      }
-      resolve(true);
+    // some versions of FF don't allow access to IndexDB in the private mode
+    let dbOpenRequest: IDBOpenDBRequest;
+
+    try {
+      dbOpenRequest = window.indexedDB.open('test');
+    } catch (error) {
+      return resolve(false);
     }
+
+    const connectionTimeout = setTimeout(() => resolve(false), 10000);
+
+    dbOpenRequest.onerror = event => {
+      clearTimeout(connectionTimeout);
+      if (dbOpenRequest.error) {
+        event.preventDefault();
+        return resolve(false);
+      }
+    };
+
+    dbOpenRequest.onsuccess = _event => {
+      clearTimeout(connectionTimeout);
+      return resolve(true);
+    };
   });
 
 const checkBrowser = (): void => {
+  if (!supportsCookies()) {
```

Comment:
Cookies are required for oAuth logic, but we cannot move it into the `basicBrowserFeatureCheck.js` as it has a dependency on `js-cookie`